sphinx/writers/latex.py

```diff
@@ -927,8 +927,9 @@ def generate(content, collapsed):
 
     def render(self, template_name, variables):
         # type: (unicode, Dict) -> unicode
-        for template_path in self.builder.config.templates_path:
-            template = path.join(template_path, template_name)
+        for template_dir in self.builder.config.templates_path:
+            template = path.join(self.builder.srcdir, template_dir,
+                                 template_name)
```

Comment:
`templates_path`s are relative path from `confdir`.
http://www.sphinx-doc.org/en/master/config.html#confval-templates_path

Reply:
Thanks, done.

Reply:
@tk0miya shouldn't https://github.com/jfbu/sphinx/blob/1.7/sphinx/config.py#L117 have been using ``['html', 'latex']`` since #3999 ? In my brief testing `make latex` after a change to `templates_path` rebuilds latex file, despite the setting being left to ``'html'``. Thus I don't understand the comment at https://github.com/jfbu/sphinx/blob/1.7/sphinx/config.py#L85

Reply:
The second value is used to determine to do rebuild or not when the config has been changed.

If the confval tagged as `env` is changed, Sphinx goes full rebuild (even if source files are not changed).
On the other hand, on the one tagged as `html` changed, Sphinx only regenerates HTML files.

But, LaTeX builder does not refer it. It always regenerate TeX file even if nothing changed.
So we should have to improve it in future. Let's reconsider at the moment.